content/post/67-crdt-research-meetup.md

```diff
@@ -0,0 +1,60 @@
+---
+date: 2019-01-28
+url: 67-crdt-research-meetup
+tags: crdt research meetup videos ipfs
+title: 2018 CRDT Research Meetup - Lisbon
+author: Pedro Teixeira
+---
+
+On April last year some members of the IPFS team gathered in Lisbon for a week of fun hack. During this week we mostly hacked and collaborated on some aspects of IPFS, libp2p and applications built on top of them, but also took one day to host a CRDT Research meetup.
```

Comment:
```suggestion
We've hosted the first Lisbon CRDT Research Meetup on April 2018. It was a brilliant event with the presence of many of the authors of the CRDT research literature and representatives from the Labs it originated, namely: [HASLab](https://haslab.uminho.pt/), [NOVA-LINCS](http://nova-lincs.di.fct.unl.pt/), [FCT](https://www.fct.unl.pt/), [UMinho](https://uminho.pt/), [UNL](https://www.unl.pt/).
```

Reply:
David's suggestion, just minus the initial " 've" - just "we hosted the first Lisbon CRDT..."